### DISCUSSION TITLE
Maybe split "merchants" into distinct companies?

### DISCUSSION THREAD

**Discussion by DoublePants** [endless-sky/endless-sky#10293] [Ideas]
I was thinking it might be cool if your reputation was tracked separately for separate commercial interests instead of there just being one "merchant" faction. For example, you could have the "Sol Transport Company" which would spawn passenger ships in the region of Sol, or "The Longhaul Teamsters Union" ships which would have higher odds of spawning around the rim. You could also have smuggling groups that might end up being mission targets or providing missions.

I'm not sure if it's possible based on the current code, but it might also be cool if the player's reputation with these groups would affect what missions they're offered. For example, if they do a lot of transport missions for the Sol Transport Company, they would begin to offer more lucrative contracts. If they do a lot of escort missions for, say "The ShipShield Conglomerate," the pay for those missions might increase. Meanwhile, attacking merchants or taking missions that are competitive in nature might reduce rep. For example, the Syndicate might not like the Longhaul Teamsters Union and might give you missions to interfere with them.

This could even eventually be extended to how ships in space might interact with you. If you're on really good terms with the Sol Transport Company and one of their escort ships sees you under attack, it may go out of its way to try to help you. If you're on bad terms with the ShipShield Conglomerate, they might laugh at you if you try to ask for help when your ship is disabled or out of fuel.

I think this would improve the general feeling that the universe is alive and full of people with different goals that you can either contribute to or interfere with, and make less unified factions feel less unified and therefore contribute to the worldbuilding.

**Comment by TheGiraffe3** [comment]
Also see #9940, though this is a slightly different use of the idea.

It would require some time writing missions, but no code changes would be required for the mission part.
The ships in space would be somewhat harder to code right, and I don't think you could do that very easily.
It definitely would make the game more interesting.

**Comment by Saugia** [comment]
Generally we keep governments as simple as possible; any specific companies or groups are reserved for mission fleets/npcs, but for the sake of the map, ease of interaction, not having to rewrite half of the game, and ease of rep relationships, governments like Merchant are meant to be generalized. Otherwise, we run into cases like Hai space where there are 10+ different governments and it gets very confusing to work with.

Also consider Pirates - In lore, there are several different Pirate groups, and you may find different ones in missions, but for the sake of the overall game simplicity, there is only one Pirate government.. We may break that up into areas, like how I did with the Quarg, since Pirates are more separated in story elements. This is a good example of having separate governments for different parts of the map. However, if we do that in human space with different companies and suddenly have 5+ different merchant factions, it can make things unwieldly very fast.

So, when referring to Merchants, it is to be assumed that any merchant at a given time may be part of some other merchant group or somewhere else on the map. Active representation and interaction with this to be done would come it play when a mission explores a specific group.
